OpenRA.Mods.Common/Traits/World/BridgeLayer.cs

```diff
@@ -46,7 +46,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			{
 				var bi = w.Map.Rules.Actors[bridge].Traits.Get<BridgeInfo>();
 				foreach (var template in bi.Templates)
-					bridgeTypes.Add(template.First, Pair.New(bridge, template.Second));
+					bridgeTypes.Add(template.First, Pair.New(bridge, (int)(100 * template.Second)));
```

Comment:
It would be nicer to upgrade the bridge rules to do this translation in the yaml instead.